src/index.json

```diff
@@ -5676,6 +5676,50 @@
                     "version": "0.1.1"
                 },
                 "sha256Digest": "16f3648054ccadb1d8e3110ac7febd238f1583b1c559cd19395096e9b7588be4"
+            },
+            {
+                "downloadUrl": "https://azurecliprod.blob.core.windows.net/cli-extensions/hpc_cache-0.1.2-py2.py3-none-any.whl",
+                "filename": "hpc_cache-0.1.2-py2.py3-none-any.whl",
+                "metadata": {
+                    "azext.isExperimental": true,
+                    "azext.isPreview": true,
```

Comment:
Why both experimenal and preview are both true? @zhoxing-ms can you take a look?

Reply:
Thanks for your reminding~ That's how the old code was written. I will submit a new PR later to modify it